torch/csrc/jit/tensorexpr/tensorexpr_init.cpp

```diff
@@ -494,10 +494,12 @@ void initTensorExprBindings(PyObject* module) {
       .def(
           "cache_accesses",
           [](LoopNest& self,
-             const Buf* producer,
+             const BufHandle& producer,
```

Comment:
Why is this fixing the error? Why can't we use `Buf*` here?

Reply:
Oh, this PR does not fix the error. It is needed to reproduce the error. Specifically, `Buf` has not been added in python bindings but `BufHandle` has. So we have to modify its API to use `cache_accesses` transformation in python.

Reply:
Aah okay. Sounds good.

Reply:
I wonder why we pass buf handles as `const BufHandle*` in other places in this file. @navahgar @Chillee, any ideas about that? Going with the reference instead of the pointer seems right to me.

Reply:
Yeah, that is definitely not the right way. We should either have `const BufHandle&` if it will not be modified or a `BufHandle&` (or even pass by value) if it will be modified.

Reply:
I'll start a PR for that.